File: src/main/java/dev/lampirg/consultationappointment/ConsultationAppointmentApplication.java
package dev.lampirg.consultationappointment;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableAsync;

@SpringBootApplication
@EnableAsync
public class ConsultationAppointmentApplication {

	public static void main(String[] args) {
		SpringApplication.run(ConsultationAppointmentApplication.class, args);
	}
}

File: src/main/kotlin/dev/lampirg/consultationappointment/ConsultationAppointmentApplication.kt
package dev.lampirg.consultationappointment

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableAsync
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EnableAsync
@EnableScheduling
class ConsultationAppointmentApplication

fun main(args: Array<String>) {
    runApplication<ConsultationAppointmentApplication>(*args)
}
